core/SCsub

```diff
@@ -80,6 +80,8 @@ if env['builtin_zlib']:
 	env_thirdparty.Prepend(CPPPATH=[thirdparty_zlib_dir])
 	# Needs to be available in main env too
 	env.Prepend(CPPPATH=[thirdparty_zlib_dir])
+	if (env['target'] != 'release'):
```

Comment:
Is it needed for `target=release_debug`?

Reply:
It the same as in `freetype` module (https://github.com/godotengine/godot/blob/master/modules/freetype/SCsub#L69) which uses these symbols.

Reply:
It should be changed in both places to `env['target'] == 'debug'`, that's the only target for which `DEBUG` is defined (and only on some platforms, apparently Windows, UWP and iOS - and I'm not sure it's done on purpose).`

Reply:
> It should be changed in both places to env['target'] == 'debug'

Done.